### DISCUSSION TITLE
Playbook sofilter deletion after update community respository

### DISCUSSION THREAD

**Discussion by sleepingbel** [Security-Onion-Solutions/securityonion#8148] [Unsupported Versions] [answered]
Hello all,

If a community rule changes after a update of the respository, will the sofilter be removed of the changed rules? 

I ask this because it is possible that the filter created for the old rule will also filter out the true positives on the updated rule.

Regards
Bart

**Comment by defensivedepth** [comment, marked as answer]
The custom filter as defined here (https://docs.securityonion.net/en/2.3/playbook.html#tuning-plays) will not be deleted after community rules are updated.

**Reply by sleepingbel** [reply]
Hello Josh,

When you update the community rules haw can you see witch rules are changed? So you can check the rules if the sofilter is still applicable.

regards 
Bart

**Reply by defensivedepth** [reply]
You can sort the updated plays by the `updated` column:

![image](https://user-images.githubusercontent.com/954732/175116432-1c835b0a-9eb8-49b1-ab74-5397838e6bbd.png)






And then you can check out the Elastalert Diffs:

![image](https://user-images.githubusercontent.com/954732/175116597-f95ce327-e066-48e8-a1db-047ec39d620d.png)

**Reply by sleepingbel** [reply]
thx
